### PR TITLE
CLI signing improvements

### PR DESCRIPTION
**Pull request by roman-khimov**
### Problem

#2662, #2664

### Solution

Solve 'em.

### CODE REVIEW

**Review by AnnaShaleva — COMMENTED** [review]
Minor typo, otherwice LGTM.

And I have a question, currently, we can't provide arguments for deployed contract signature (if it requires parameters). Can't we extend existing signing interface to work with deployed contract with parameters? It would be nice to have it, maybe create a separate issue?

File: cli/wallet/wallet.go [hunk 1]
```diff
@@ -292,6 +292,23 @@ func NewCommands() []cli.Command {
...
 			},
+			{
+				Name:      "strip-keys",
+				Usage:     "remove private keys for all accounts",
+				UsageText: "neo-go wallet strip-keys -w wallet [--wallet-config path] [--force]",
+				Description: `Removes private keys for all accounts from the given wallet. Notice,
+   this is a very dangerous action (you can lose keys if you don't have a wallet
+   backup) that should not be performed unless you know what you're doing. It's
+   mostly useful for creation of special wallets that can be used to create
+   transactions, but can't be used to sign them (offline signing).
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply cli/wallet/wallet.go:303]: The documentation made it clear.

**Comment by roman-khimov** [comment]
> we can't provide arguments for deployed contract signature (if it requires parameters)

Deployed contracts have them in the manifest, so it should be used if needed. The bigger problem is custom contracts (I can have anything in my verification script) where parameters do need to be specified manually, there is no way around that. An issue, please (not a top priority, but can be done some day).

File: pkg/smartcontract/context/context_test.go [hunk 2]
```diff
@@ -96,24 +96,38 @@ func TestParameterContext_AddSignatureMultisig(t *testing.T) {
...
+			v := newTestVM(w, tx)
+			require.NoError(t, v.Run())
+			require.Equal(t, 1, v.Estack().Len())
+			require.Equal(t, true, v.Estack().Pop().Value())
+		})
+	}
+	t.Run("exact number of sigs", func(t *testing.T) {
+		testSigWit(t, 3)
+	})
+	t.Run("exact number of sigs", func(t *testing.T) {
```
[the remark this answers is not in the document]
Comment (roman-khimov) [reply pkg/smartcontract/context/context_test.go]: Fixed.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**